multinet/uploaders/csv.py

```diff
@@ -95,6 +101,10 @@ def upload(
 
     # Create table and insert the data
     loaded_table = loaded_workspace.create_table(table, edges)
+
+    if metadata:
+        loaded_table.set_metadata(json.loads(metadata))
```

Comment:
This will raise an exception if you pass non-json as the metadata argument. Should we catch it here and turn it into a 4xx error?

Reply:
Yep